activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/rest/ProcessInstanceRelProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.query.rest;
+
+import org.activiti.cloud.services.query.model.Variable;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.hateoas.RelProvider;
+import org.springframework.stereotype.Component;
+
+@Component
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class ProcessInstanceRelProvider implements RelProvider {
+
+    public static final String COLLECTION_RESOURCE_REL = "process-instances";
+
+    private static final String ITEM_RESOURCE_REL = "process-instance";
+
+    @Override
+    public String getItemResourceRelFor(Class<?> aClass) {
+        return ITEM_RESOURCE_REL;
+    }
+
+    @Override
+    public String getCollectionResourceRelFor(Class<?> aClass) {
+        return COLLECTION_RESOURCE_REL;
+    }
+
+    @Override
+    public boolean supports(Class<?> aClass) {
+        return Variable.class.isAssignableFrom(aClass);
```

Comment:
That's probably wrong. Shouldn't be `ProcessInstance` instead?

Reply:
oh i am going to fix that..

Reply:
pushed the change now